docs/entities/tokens/fungible-token.schema.example.json

```diff
@@ -7,5 +7,7 @@
   "tx_id": "0xef2ac1126e16f46843228b1dk4830e19eb7599129e4jf392cab9e65ae83a45c0",
   "sender_address": "ST399W7Z9WS0GMSNQGJGME5JAENKN56D65VGMGKGA",
   "symbol": "HEY",
-  "decimals": 5
+  "decimals": 5,
+  "microblock_hash": "0x",
+  "microblock_sequence": "-1"
```

Comment:
I don't think adding these microblock properties to the response is helpful. Maybe if we also added the block hash and block height then it would make sense, otherwise should probably leave out of this PR.